pkg/pool-weighted/test/ManagedPool.test.ts

```diff
@@ -388,6 +388,28 @@ describe('ManagedPool', function () {
             bptAmount: actualManagementFeeBpt,
           });
         });
+
+        it('syncs the total supply to the actual supply', async () => {
+          // As we're performing a join or exit here we need to account for the change in the BPT total supply due to
+          // the join/exit. We do this by tracking the user's balance.
+          const balanceBefore = await pool.balanceOf(other);
+          const totalSupplyBefore = await pool.totalSupply();
+          const expectedManagementFeeBpt = expectedAUMFees(totalSupplyBefore, managementAumFeePercentage, timeElapsed);
+
+          const expectedActualSupply = totalSupplyBefore.add(expectedManagementFeeBpt);
+          const actualSupplyBefore = await pool.getActualSupply();
+          expect(actualSupplyBefore).to.be.equalWithError(expectedActualSupply, 0.0001);
```

Comment:
The fees are likely very small. Can we decrease the relative error here, or even test for strict equality?